packages/feather_async/feather_async.0.1.0/opam

```diff
@@ -0,0 +1,32 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "b1fb1b724cc0"
+synopsis:
+  "Async interface to Feather, a library for executing shell commands easily"
+maintainer: ["charles@nrwhl.xyz"]
+authors: ["Charles Chamberlain"]
+license: "MIT"
+depends: [
+  "dune" {>= "2.6"}
+  "base"
+  "async"
+  "feather"
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
```

as above. You may also need to add ppx_expect as dependency since it is used in tests